Library/Homebrew/env_config.rb

```diff
@@ -228,6 +228,18 @@ module EnvConfig
                      "of Ruby is new enough.",
         boolean:     true,
       },
+      HOMEBREW_FORMULA_BUILD_NETWORK:            {
+        description: "If set, controls network access to the sandbox for formulae builds. " \
```

Comment:
Unclear what the default behaviour is without this set. May also be worth mentioning if this overrides the DSL usage.